apps/oxlint/src/output_formatter/mod.rs

```diff
@@ -111,3 +111,53 @@ impl OutputFormatter {
         self.internal.get_diagnostic_reporter()
     }
 }
+
+#[cfg(test)]
+mod test {
+    use crate::tester::Tester;
+
+    const TEST_CWD: &str = "fixtures/output_formatter_diagnostic";
+
+    #[test]
+    fn test_output_formatter_diagnostic_default() {
+        let args = &["--format=default", "test.js"];
+
+        Tester::new().with_cwd(TEST_CWD.into()).test_and_snapshot(args);
+    }
+
+    /// disabled for windows
+    /// json will output the offset which will be different for windows
+    /// when there are multiple lines (`\r\n` vs `\n`)
+    #[cfg(all(test, not(target_os = "windows")))]
```

Comment:
Maybe there's a configuration somewhere in the json formatter that can eliminate different line endings.

Reply:
It is the offset of the analyzed file. The byte length differs when opening under windows. 
I do not know how to solve this because its a "problem" inside oxc_linter/miette.